### DISCUSSION TITLE
Antivirus deleted Win64/GenKryptik.FXIE

### DISCUSSION THREAD

**Discussion by xurshid29** [community/community#20268] [Copilot Conversations]
Hello, my antivirus (ESET NOD32) detected this file as threat (Win64/GenKryptik.FXIE) and deleted it. Is it a safe file? Do I have to add it to the exception list?

![Screenshot 2022-07-10 103123](https://user-images.githubusercontent.com/5887811/178132755-3d660e5e-b72d-4bfc-9524-77e47ce75e94.png)

**Comment by radar-blip** [comment]
According to Microsoft. A virus with very similiar name can be detected and removed with Microsoft Defender. The virus seems to be some kind of trojan malware/ransomware which you probably dont want on your system.
Link here: https://www.microsoft.com/en-us/wdsi/threats/malware-encyclopedia-description?Name=Trojan:Win32/GenKryptik
Another: https://cleanpcinfections.com/2022/02/how-can-i-remove-win32-genkryptik-fqfw/
Here is the link that describes it as ransomware (note the difference in the extension): https://howtofix.guide/win64-genkryptik-fnts/

It doesn't sound safe, I would get rid of it completely if possible.
Anyway, Good luck!!

**Reply by Chavk77** [reply]
Най добре е да са отърва от него

**Reply by xurshid29** [reply]
This file is coming from the "github-copilot-intellij"
![Screenshot 2022-07-10 133415](https://user-images.githubusercontent.com/5887811/178137561-806b3166-badd-4de4-9cc0-5f76cf9c3d0c.png)

**Reply by radar-blip** [reply]
This member seems to be having the same issue:  #20247
